install/helm/gloo/generate/values.go

```diff
@@ -645,15 +657,18 @@ type Mtls struct {
 }
 
 type SdsContainer struct {
-	Image *Image `json:"image,omitempty"`
+	Image           *Image                  `json:"image,omitempty"`
+	SecurityContext *appsv1.SecurityContext `json:"securityContext,omitempty" desc:"securitySpec for sds gloo deployment container If this is defined it supercedes any values set in FloatingUserId, RunAsUser, DisableNetBind, RunUnprivileged. https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.26/#securitycontext-v1-core"`
```

Comment:
```suggestion
	SecurityContext *appsv1.SecurityContext `json:"securityContext,omitempty" desc:"SecurityContext for sds gloo deployment container. If this is defined it supercedes any values set in FloatingUserId, RunAsUser, DisableNetBind, RunUnprivileged. See https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.26/#securitycontext-v1-core for details."`
```
added punctuation / full sentence. can you apply similar changes to other instances below?

Reply:
Done as a batch